_tests/browser_tests/.eslintrc

```diff
@@ -0,0 +1,10 @@
+env:
+  node: true
```

Comment:
I suppose though if this is a required setting it should be set to true here as well. That way if for some reason the main eslintrc file was updated it wouldn't break here. :+1: